iaso/tests/fixtures/odk_invalid_xlsform_expected_errors.json

```diff
@@ -1006,18 +1006,18 @@
         "severity": "error"
     },
     {
-        "message": "choices not for 'nonexistingchoices' and list_name 'doenst_exist'",
+        "message": "choices not found for 'nonexistingchoices' and list_name 'doenst_exist'",
         "question": {
-            "line_number": 98,
-            "type": "calculate",
-            "name": "without_known_ref",
-            "label": "",
+            "line_number": 53,
+            "type": "select_one doenst_exist",
+            "name": "nonexistingchoices",
+            "label": "This should throw a warning there\u2019s no associated choice",
```

Comment:
I guess `\u2019s` is used on purpose here, or is it a typo?

Reply:
it's the output of the "json.dumps" so I guess it escape these by default.